geoutils/spatial_tools.py

```diff
@@ -136,8 +137,8 @@ def load_multiple_rasters(
 
     # Optionally, crop the rasters
     if crop:
-        # Check that intersection is not void
-        if intersection == ():
+        # Check that intersection is not void (changed to NaN instead of empty tuple end 2022)
+        if intersection == () or all(math.isnan(i) for i in intersection):
```

Comment:
What's the advantage of  `math.isnan` to `~np.isfinite`?

Reply:
Hmmm, curious, for some reason when I tried with `np.isnan` the first time, it did not work, because the output of geopandas is a `float('nan')`, but now it works element-wise...
I changed to `np.isnan`.